frontend/src/components/lib/Gallery/index.jsx

```diff
@@ -18,17 +19,63 @@ const Gallery = React.forwardRef(function Gallery(
   ];
   const { t } = useTranslation('gallery');
 
+  const [ galleryIsOpen, setGalleryIsOpen ] = React.useState(false);
+  const [ secondGalleryIsOpen, setSecondGalleryIsOpen ] = React.useState(false);
+
+  const [ pictureIndex, setPictureIndex ] = React.useState(0);
+
+  const openGallery = () => {
+    setGalleryIsOpen(true);
+  };
+
+  const closeGallery = () => {
+    setGalleryIsOpen(false);
+  };
+
+  const openSecondGallery = (index) => {
+    setSecondGalleryIsOpen(true);
+    setPictureIndex(index);
+  };
+
+  const closeSecondGallery = () => {
+    setSecondGalleryIsOpen(false);
+  };
+
   return (
     <div ref={ref} className={styles.Gallery} {...delegated}>
 
       {pictures.map((picture, index) => (
         index > 4 ? null :
           <div key={index} className={styles[gridClassArray[index]]}>
             {picture}
-            {index === 4 && pictures.length > 5 ? <Button href="javascript:void" className={styles.GalleryMoreBtn}>{t('viewPhotos', { count: pictures.length })}</Button> : null}
+            {index === 4 && pictures.length > 5 ? <Button href="javascript:void" onClick={openGallery} className={styles.GalleryMoreBtn}>{t('viewPhotos', { count: pictures.length })}</Button> : null}
```

Comment:
```suggestion
            {index === 4 && pictures.length > 5 ? <Button variant="secondary" onClick={openGallery} className={styles.GalleryMoreBtn}>{t('viewPhotos', { count: pictures.length })}</Button> : null}
```

Utilise le type secondary il n'y a pas de background